### PR TITLE
Add pyGLFW

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyglfw/run_tests.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+
+# Ensure the GLFW library can be found.
+CONDA_PREFIX = os.environ['CONDA_PREFIX']
+
+if sys.platform == 'linux':
+    os.environ['LD_LIBRARY_PATH'] = CONDA_PREFIX + os.path.sep + 'lib'
+elif sys.platform == 'darwin':
+    os.environ['DYLD_LIBRARY_PATH'] = CONDA_PREFIX + os.path.sep + 'lib'
```
Comment (scopatz) [inline recipes/pyglfw/run_tests.py]: if you need to do this, something is wrong

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: Hm ok? I didn't find another way to fix this, though…

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: Would that be a problem with the `glfw` package, then?

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: @scopatz  Wonder if this could be a solution?
https://github.com/conda/conda/issues/1533#issuecomment-132737669

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: `pyGLFW` will also check the `PYGLFW_LIBRARY` path variable to locate the `glfw` library. Would it be possible to modify the `glfw` package to set this variable automatically once the respective `conda` environment is being activated?

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: Seems to be an ongoing problem with numerous libs:
https://github.com/ContinuumIO/anaconda-issues/issues/1716

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: @scopatz This needs to be fixed upstream, no?
https://github.com/FlorianRhiem/pyGLFW/blob/master/glfw/library.py#L123-L144

An alternative could be to [set `PYGLFW_LIBRARY`](https://github.com/FlorianRhiem/pyGLFW/blob/master/glfw/library.py#L147-L149), but I don't know how one could do that automatically one the environment gets activated?

Comment (hoechenberger) [reply recipes/pyglfw/run_tests.py]: I've included a patch.

**Comment by hoechenberger** [comment]
@scopatz Ok with the now-included patch, this working fine. I'm not sure what's wrong with the Windows builds though??

**Comment by hoechenberger** [comment]
Files a PR upstream to eventually make the patch superfluous:
https://github.com/FlorianRhiem/pyGLFW/pull/30

**Comment by hoechenberger** [comment]
@kastman Could you please confirm you're okay with being a maintainer of this recipe?

**Comment by hoechenberger** [comment]
@scopatz All good from my side.

**Comment by kastman** [comment]
Yes, happy to be a maintainer

**Pull request merged**